pkgs/by-name/hi/highctidh/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  python3Packages,
```

Comment:
We should remove `python3Packages` and explicitly call every dependency, here. Example:

```nix
{
  lib,
  buildPythonPackage,
  setuptools,
  ...
}:
```